docs/book/src/quinn/certificate.md

```diff
@@ -0,0 +1,163 @@
+# Certificates
+
+In this chapter, we discuss the configuration of the certificates that is **required** for a working Quinn connection. 
+
+QUIC uses TLS 1.3 for authentication of connections, the server will have to provide the client with a certificate confirming its identity, 
+and the customer must be configured to trust the certificates he receives from our server. 
```

Comment:
The difference between "client" and "customer" seems confusing here. The connection between "QUIC uses TLS 1.3" and "the server will have" also seems to be left out.

Reply:
Ah I think its a translator issue here.